pkg/auth/authenticator/password/basicauthpassword/basicauthpassword.go

```diff
@@ -52,6 +52,13 @@ type RemoteError struct {
 	Error string
 }
 
+// Error type indicating the type of redirect that was detected
+type RedirectAttemptedError string
```

Comment:
can just make `var RedirectAttempted = errors.New("redirect attempted")` and return it directly